tempesta_fw/str_simd.c

```diff
@@ -143,6 +146,13 @@ tfw_str_init_const(void)
 		0xf8, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc,
 		0xfc, 0xfc, 0xfc, 0xf4, 0xd8, 0xfc, 0xfc, 0x7c);
 
+	/* ASCII zero character '\0' to use in strict signed comparison. */
```

Comment:
It seems that it is not ASCII zero or `\0`, but `0x80` value.

Reply:
`0x80` is used to go from necessary unsigned comparison to signed comparison which we only have in SIMD. I described the step from Hacker's Delight in the blog post http://natsys-lab.blogspot.ru/2016/10/http-strings-processing-using-c-sse42.html . But, yes, `0 - 0x80` would be simpler and more readable.